laze-project.yml

```
@@ -388,6 +390,7 @@ modules:
           # For some reason, `sccache` makes the build not realize changes to
           # `DEFMT_LOG`. Painful as it is, hard-disable `sccache` here.
           - RUSTC_WRAPPER=""
+          - DEFMT_LOG=${LOG}
```

```suggestion
          - DEFMT_LOG=info,${LOG}
```

Default to `info`? If the user also sets `-DLOG=<level>` it would simply overwrite this.

---

this does not seem to work?

---

I just tested, it works for me:

```sh
$ laze build  -C examples/log -b rpi-pico run
...
    Finished `release` profile [optimized + debuginfo] target(s) in 1.54s
     Running `probe-rs run --chip RP2040 ../../build/bin/rpi-pico/log/cargo/thumbv6m-none-eabi/release/log`
      Erasing ✔ [00:00:00] [###########################################################] 20.00 KiB/20.00 KiB @ 65.59 KiB/s (eta 0s )
  Programming ✔ [00:00:00] [###########################################################] 20.00 KiB/20.00 KiB @ 32.84 KiB/s (eta 0s )    Finished in 0.951s
INFO  info log level enabled
WARN  warn log level enabled
ERROR error log level enabled (just testing)
riot_rs_rt::startup()
rpi-pico::init()
riot-rs-embassy::init(): using interrupt mode executor
riot-rs-embassy::init_task()
riot-rs-embassy::init_task() done
^C

$ laze build -DLOG=debug -C examples/log -b rpi-pico run
...
    Finished `release` profile [optimized + debuginfo] target(s) in 1.55s
     Running `probe-rs run --chip RP2040 ../../build/bin/rpi-pico/log/cargo/thumbv6m-none-eabi/release/log`
      Erasing ✔ [00:00:00] [###########################################################] 20.00 KiB/20.00 KiB @ 65.98 KiB/s (eta 0s )
  Programming ✔ [00:00:00] [###########################################################] 20.00 KiB/20.00 KiB @ 32.01 KiB/s (eta 0s )    Finished in 0.962s
riot_rs_rt::startup()
rpi-pico::init()
riot-rs-embassy::init(): using interrupt mode executor
riot-rs-embassy::init_task()
riot-rs-embassy::init_task() done
DEBUG debug log level enabled
INFO  info log level enabled
WARN  warn log level enabled
ERROR error log level enabled (just testing)
```

---

thanks for checking, it works here, too. on the nucleo-wb55, it needs a reboot more, the first "run" after changing the log level doesn't show any defmt output. :/